crypto/keys/multisig/amino.go

```diff
@@ -78,7 +78,9 @@ func (m *LegacyAminoPubKey) UnmarshalAminoJSON(tmPk tmMultisig) error {
 	// Instead of just doing `*m = *protoPk`, we prefer to modify in-place the
 	// existing Anys inside `m` (instead of allocating new Anys), as so not to
 	// break the `.compat` fields in the existing Anys.
+	m.PubKeys = make([]*types.Any, len(protoPk.PubKeys))
 	for i := range m.PubKeys {
+		m.PubKeys[i] = &types.Any{}
```

Comment:
IMO this fix is not correct. Reading the comment just above, we actually don't want to create new Anys, because we want to keep the existing Anys' `.compat` field.

The unmarshalling error comes from the fact that `.compat` is empty on these Anys.